test/tableau/extensions/post_extension_test.exs

```diff
@@ -85,6 +85,18 @@ defmodule Tableau.PostExtensionTest do
       Do cars fly yet?
       """)
 
+      File.write(Path.join([dir, "_drafts", "my-post-in-drafts-dir.md"]), """
```

Comment:
i'm not sure this extra fixture is adding anything to the test

Reply:
That test has a negative case where a future post isn't included in `posts`, so I was attempting to create a similar negative case for posts in a different folder. I don't think I constructed the case correctly.

I will remove it for now, because I don't think it's functioning correctly, but I can write that test if you think it's valuable.